### ISSUE TITLE
can't reproduce and got noise issue 

### ISSUE THREAD

**Issue by jnkr36** [LambdaLabsML/examples#33]
**TRAIN:** I follow the example and use V100 to reproduce, I just change the batch size from 4 to 1 in configs/stable-diffusion/pokemon.yaml
python main.py -t --base configs/stable-diffusion/pokemon.yaml --gpus 1 --scale_lr False --num_nodes 1 --check_val_every_n_epoch 10 --finetune_from sd-v1-4-full-ema.ckpt

**TEST**: After training about 300 epochs, I use scripts/txt2img.py to test:
1、first I use the original checkpoint sd-v1-4-full-ema.ckpt to test and get the below result:
python scripts/txt2img.py --prompt 'robotic cat with wings' --outdir '/outputs/generated_pokemon' --H 512 --W 512 --n_samples 4 --config '/configs/stable-diffusion/pokemon.yaml' --ckpt 'sd-v1-4-full-ema.ckpt'
![image](https://user-images.githubusercontent.com/36756751/198968831-34b5edd2-2ce0-4691-b6e2-6a911b032c66.png)
2、and then I use epoch=000002.ckpt、epoch=000004.ckpt、epoch=000007.ckpt、epoch=000009.ckpt、epoch=0000012.ckpt......to test again, and the result becomes more and more like noise, and at last i only generate all black picture.
 python scripts/txt2img.py --prompt 'robotic cat with wings' --outdir '/outputs/generated_pokemon' --H 512 --W 512 --n_samples 4 --config '/configs/stable-diffusion/pokemon.yaml' --ckpt 'logs/2022-10-28T12-32-02_pokemon/checkpoints/epoch=000002.ckpt'
1)epoch=000002.ckpt result:
![image](https://user-images.githubusercontent.com/36756751/198969019-d92875e8-8807-43de-b745-821b2bd3c0bb.png)
2)epoch=000004.ckpt result:
![image](https://user-images.githubusercontent.com/36756751/198969091-9ecf0da3-9639-40e5-b39c-12940dc50756.png)
3)epoch=000007.ckpt result:
![image](https://user-images.githubusercontent.com/36756751/198969431-a6705843-8c91-404b-ba22-843e4ced897d.png)
4)epoch=000009.ckpt result:
![image](https://user-images.githubusercontent.com/36756751/198964851-b4d2569e-d0c4-4be4-b65d-1784f06a0a1f.png)
5)epoch=000012.ckpt result:
![image](https://user-images.githubusercontent.com/36756751/198964915-03a456f6-cc58-45d0-854f-c5cc3b455836.png)
6)epoch=000014.ckpt result:
![image](https://user-images.githubusercontent.com/36756751/198965026-3fa9b9ba-baf9-40f1-9791-305d29a76395.png)
......
7)epoch=000048.ckpt result:
![image](https://user-images.githubusercontent.com/36756751/198965103-ff87d9f4-3222-4fc7-b540-8460d1fc489f.png)
Is there anyone meet the same issue? or could you someone help to solve the problem?

**Comment by lk-wq** [comment]
You mentioned you set the batch size from 4 to 1, did you also scale the learning rate down?

**Comment by jnkr36** [comment]
> You mentioned you set the batch size from 4 to 1, did you also scale the learning rate down?

Thanks so much for your suggestion. I scale base_learning_rate in configs/stable-diffusion/pokemon.yaml from 1.0e-04 to 2.0e-05 and try again, it seems that it can work now.
Currently, training process has run to epoch 129, I will run it for more time and hope to get better results
1）epoch=000052.ckpt result:
![image](https://user-images.githubusercontent.com/36756751/199401186-44718fb6-ee4f-49b8-96a0-6a48bc06e8b3.png)

2）epoch=000100.ckpt result:
![image](https://user-images.githubusercontent.com/36756751/199415441-832848b7-f1c4-45c7-a770-0ba3cba1093e.png)

3）epoch=000129.ckpt result:
![image](https://user-images.githubusercontent.com/36756751/199390416-c03df98a-16ee-4891-b041-643cc2f26bde.png)

**Comment by tenpha** [comment]
> 

Wow, I also encontered this problem. Thanks!

**Comment by xuzekai1997** [comment]
hello, could you share your yaml file.When I run main.py, it always got wrong.But I have no idea of which part is wrong.
Thanks a lot.

**Comment by micklexqg** [comment]
> > You mentioned you set the batch size from 4 to 1, did you also scale the learning rate down?
> 
> Thanks so much for your suggestion. I scale base_learning_rate in configs/stable-diffusion/pokemon.yaml from 1.0e-04 to 2.0e-05 and try again, it seems that it can work now. Currently, training process has run to epoch 129, I will run it for more time and hope to get better results 1）epoch=000052.ckpt result: ![image](https://user-images.githubusercontent.com/36756751/199401186-44718fb6-ee4f-49b8-96a0-6a48bc06e8b3.png)
> 
> 2）epoch=000100.ckpt result: ![image](https://user-images.githubusercontent.com/36756751/199415441-832848b7-f1c4-45c7-a770-0ba3cba1093e.png)
> 
> 3）epoch=000129.ckpt result: ![image](https://user-images.githubusercontent.com/36756751/199390416-c03df98a-16ee-4891-b041-643cc2f26bde.png)

@jnkr36 ,Hi, how you solved it, only scale the learning rate down when batch size is changed?
